ci/root/etc/docker/daemon.json

```diff
@@ -0,0 +1,3 @@
+{
+  "registry-mirrors": ["http://registrycache:5000"]
```

Comment:
How is registrycache service setup? I don't see any reference except here in that PR.

Reply:
The registrycache is another separate container that needs to be run, if the local docker service cannot hit the registry cache it will automatically go to DockerHub as if the cache did not exist, so it's transparent.

I will provide a `docker-compose.yml` which includes strings for both the cache and the respective nodes.
There's also some files you will need to map in to the buildkite node containers which I will address in the `README.md`.

Reply:
I've just added an example `docker-compose.yml` and a `README.md` on usage.